Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.ts

```diff
@@ -124,40 +126,17 @@ export class ProfilePictureComponent implements OnInit, OnDestroy {
     if (this.canEnlargeOnClick === false || this.dialogRef != null || this.imageLoaded === false) {
       return;
     }
-    this.openEnlarge(true);
-  }
-
-  public onMouseleave() {
-    if (this.canEnlargeOnHover === false) {
-      return;
-    }
-    this.closeEnlarge();
+    this.openEnlarge();
   }
 
-  public openEnlarge(backdrop: boolean = false) {
+  public openEnlarge() {
     let url = this.imageUrl;
+    const altText = this.altText;
     if (!this.useAprilEndpoint) {
       url = `${runtimeEnvironment.employeePicturesEndpoint}/generated/${this.user.Id}/900.jpg`;
     }
-    this.dialogRef = this.dialog.open(ProfilePictureEnlargeDialog, {
-      hasBackdrop: backdrop,
-      data: {
-        imageUrl: url,
-        text: this.altText,
-      },
-    });
-    this.dialogRef.afterClosed().subscribe((closed) => {
-      this.dialogRef = null;
-    });
-  }
-
-  public closeEnlarge(): void {
-    if (this.dialogRef != null) {
-      this.dialogRef.close();
-    }
-  }
 
-  public ngOnDestroy() {
-    this.closeEnlarge();
+    this.dialogService.displayDialog('profile-picture', { url, altText });
```

Comment:
does this work? I think the object constructed will be 
```json
{
  url: url,
  altText: altText
}
```
instead of
```json
{	
   imageUrl: url,	
   text: this.altText,	
}
```

Reply:
I don't know if that'll work as I said in the PR description. I need to figure out a way to log into the backend as a User so I can access the Profile Page.
May you could help me out with that @DanielHabenicht ?

Reply:
Go into the runtime environment settings and change the backend and picture URL accordingly. Write @friedaxvictoria in rocket chat she did it recently.